data/state/pubkeyConverter/bech32PubkeyConverter.go

```diff
@@ -71,19 +75,34 @@ func (bpc *bech32PubkeyConverter) Bytes(humanReadable string) ([]byte, error) {
 	return decodedBytes, nil
 }
 
-// String converts the provided bytes in a bech32 form
-func (bpc *bech32PubkeyConverter) String(pkBytes []byte) (string, error) {
-	conv, err := bech32.ConvertBits(pkBytes, 8, 5, bech32Config.pad)
+// Encode converts the provided bytes in a bech32 form
+func (bpc *bech32PubkeyConverter) Encode(pkBytes []byte) string {
+	//since the errors generated here are usually because of a bad config, they will be treated here
```

Comment:
the pkBytes from the Encode come from the tx.Sender and tx.Receiver and are not necessarily from a bad config. With invalid pkBytes the method would return "" and this would be later used. Is it ok?

Reply:
yes, if you provide garbage, you get garbage, but the code does not necessarily break (same as hex.Encode). In the case of faulty transaction, that transaction is verified for correctness whenever it reaches the node. Not necessarily to check again in this implementation.

Reply:
ok